test/integration/rules/image-alt/image-alt.html

```diff
@@ -16,3 +16,4 @@
 <img src="img.jpg" id="pass8" aria-labelledby="ninjamonkeys">
 <div role="img" alt="blah" id="violation6"></div>
 <div role="img" aria-label="blah" id="pass9"></div>
+<svg rol="img" id="ignore1"><title>SVG Title</title></svg>
```

Comment:
There is a typo in the attribute

Reply:
Please fix the typo. I think you're right about excluding svg from this rule. We should have a separate SVG rule instead.